### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Generated by https://github.com/jsoref/spelling `f`; to maintain your repo, please consider `fchurn`

Note: there's at least one API break in here.

### CODE REVIEW

File: agent/kvs_endpoint.go [hunk 1]
```diff
@@ -95,8 +95,8 @@ func (s *HTTPServer) KVSGetKeys(resp http.ResponseWriter, req *http.Request, arg
 	// we now are forced to check for both spellings
 	var sep string
 	params := req.URL.Query()
-	if _, ok := params["seperator"]; ok {
-		sep = params.Get("seperator")
+	if _, ok := params["separator"]; ok {
```
Comment (jsoref) [inline agent/kvs_endpoint.go]: This is an API break.

Comment (mitchellh) [reply agent/kvs_endpoint.go]: Heh, we're aware of this mistake. Unfortunately, it shipped and we can't break compatibility here so if all of these related changes can be reverted that'd be very helpful.

Comment (jsoref) [reply agent/kvs_endpoint.go]: Dropping that is trivial (it's a single commit)

**Comment by jsoref** [comment]
@mitchellh: dropped

File: CHANGELOG.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ BUG FIXES:
 
 * agent: (Consul Enterprise) Fixed an issue where the snapshot agent's HTTP client config was being ignored in favor of the HTTP command-line flags.
```
Comment (pearkes) [inline CHANGELOG.md]: Given it is a historical record I recommend we drop changes to `CHANGELOG.md`, less of a living document.

**Comment by jsoref** [comment]
@pearkes: dropped

**Pull request merged**